src/theme.ts

```diff
@@ -83,6 +83,12 @@ const theme = createMuiTheme({
         body: {
           backgroundColor: '#05699b',
         },
+        h6: {
+          fontSize: '16px',
+          color: '#c4c4c4',
+          fontWeight: 600,
+          letterSpacing: '0.15px',
+        },
```

Comment:
I think we should use the Typography component. https://material-ui.com/api/typography/ Maybe we could setup these styles in the theme if they don't match the defaults.

@j2trumpet you have any thoughts/want to help with this?

Reply:
I created this to use the style that was created in Figma which I think is used in multiple places.  @j2trumpet I would love to know your thoughts on this.

![MPDX_Update_Material_UI](https://user-images.githubusercontent.com/970048/113335966-b1a4ae00-92f3-11eb-842a-e74704bdf727.png)

Reply:
using the typography component you do this:
```
<Typography variant="h6">Text Content</Typography>
```

it will then automatically pull the h6 styles from the MUI theme

Reply:
I don't know if this is or isn't the right spot to specify that, I would assume that it might inherit some of these settings automatically from the base MUI theme.